axelor-account/src/main/resources/views/AccountingBatch.xml

```diff
@@ -360,10 +360,7 @@
 
   <action-record name="action-record-accounting-batch-journal-onchange"
     model="com.axelor.apps.account.db.AccountingBatch">
-    <field name="cutOffMoveStatusSelect" expr="eval: __repo__(Move).STATUS_SIMULATED"
-      if="miscOpeJournal?.authorizeSimulatedMove"/>
-    <field name="cutOffMoveStatusSelect" expr="eval: __repo__(Move).STATUS_DAYBOOK"
-      if="!miscOpeJournal?.authorizeSimulatedMove &amp;&amp; miscOpeJournal?.allowAccountingDaybook"/>
+    <field name="cutOffMoveStatusSelect" expr="eval: null"/>
```

Comment:
Can you keep what you deleted and set to null cutOffMoveStatusSelect only if the 2 first ifs are not true?